packages/react-heartwood-components/src/components/Button/Button-story.js

```diff
@@ -88,7 +79,6 @@ stories
 	))
 	.add('Caution', () => (
 		<Button
-			isSmall={isSmall}
```

Comment:
@gcpantazis curious why these got removed. looks like it was using the boolean knob to allow `isSmall` to be toggled on or off in storybook.

Reply:
nvm, saw the resolved comment from carolyn above!